query-engine/driver-adapters/executor/src/bench.ts

```diff
@@ -1,244 +1,269 @@
-/**
- * Run with: `node --experimental-wasm-modules ./example.js`
- * on Node.js 18+.
- */
-import { webcrypto } from "node:crypto";
-import * as fs from "node:fs/promises";
-import path from "node:path";
+import * as fs from 'node:fs/promises'
+import path from 'node:path'
 import { __dirname } from './utils'
 
-import * as qe from "./qe";
+import * as qe from './query-engine'
 
-import { pg } from "@prisma/bundled-js-drivers";
-import * as prismaPg from "@prisma/adapter-pg";
-import { bindAdapter, DriverAdapter } from "@prisma/driver-adapter-utils";
+import { pg } from '@prisma/bundled-js-drivers'
+import * as prismaPg from '@prisma/adapter-pg'
+import { bindAdapter, DriverAdapter } from '@prisma/driver-adapter-utils'
 
-import { recording } from "./recording";
-import { nextRequestId } from "./requestId";
-import prismaQueries from "../bench/queries.json";
+import { recording } from './recording'
+import { nextRequestId } from './requestId'
+import prismaQueries from '../bench/queries.json'
 
-import { run, bench, group, baseline } from "mitata";
+import { baseline, bench, group, run } from 'mitata'
 
-import { QueryEngine as WasmBaseline } from "query-engine-wasm-baseline";
+import { QueryEngine as WasmBaseline } from 'query-engine-wasm-baseline'
 
 // `query-engine-wasm-latest` refers to the latest published version of the Wasm Query Engine,
 // rather than the latest locally built one. We're pulling in the Postgres Query Engine
 // because benchmarks are only run against a Postgres database.
-import { QueryEngine as WasmLatest } from "query-engine-wasm-latest/postgresql/query_engine.js";
-
-if (!global.crypto) {
-  (global as any).crypto = webcrypto;
-}
```

Comment:
Is this deletion intentional?
There's an errors in the tests that might be caused by this being deleted:
```
RuntimeError: unreachable
    at query_engine_wasm.wasm.<ahash::random_state::RandomState as core::default::Default>::default::hd692d3c3da9505be (wasm://wasm/query_engine_wasm.wasm-009fbe36:wasm-function[550]:0x120238)
```